src/clarity-angular/data/_tables.clarity.scss

```diff
@@ -22,12 +22,19 @@ $clr-table-lineheightPaddingShim: (1rem - $clr-table-lineheight)/2 !default;
 $clr-table-topcellpadding: $clr-table-cellpadding - $clr-rem-1px !default;
 $clr-table-bottomcellpadding: $clr-table-cellpadding - $clr-rem-1px !default;
 
+$clr-table-topcellpadding-cmp: $clr-table-cellpadding/2 - $clr-rem-1px !default;
+$clr-table-bottomcellpadding-cmp: $clr-table-cellpadding/2 - $clr-rem-1px !default;
+
 // Border radius of corner cells needs to be slightly less than the table's,
 // To make sure they correctly cover the whole area up to the border.
 $clr-table-cornercellradius: calc(#{$clr-default-borderradius} - 1px) !default;
 
+$table--compact-rowHeight: 1rem !default;
```

Comment:
Not important but should the variables be prefixed with `clr-` like we do for other variables in this file?

Reply:
I will address this.